src/components/input/index.tsx

```diff
@@ -126,6 +127,7 @@ export class SmoothlyInput implements Clearable, Input, Editable {
 		this.smoothlyInputLoad.emit(() => {
 			return
 		})
+		this.readonly && (this.uneditable = this.readonly)
```

Comment:
why are we setting `this.uneditable = this.readonly `?

Reply:
Its to remember if the input was readonly at first, if so the input wont change its readonly-ness when the form does (also this has been updated in a new commit)